### PR TITLE
[MODGOBI-160] - Created mod-gobi schemas

### PR DESCRIPTION
**Pull request by Dmitriy-Butramyou**
## Purpose
Create mod-gobi schemas

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Examples exist for all schemas
  - [x] Descriptions exist for all schema properties
  - [x] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-gobi/exemples/folio_order_fields.sample [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (SerhiiNosko) [inline mod-gobi/exemples/folio_order_fields.sample:1]: typo: please rename to examples

**Review by SerhiiNosko — APPROVED** [review]

**Comment by dcrossleyau** [comment]
Attention to @Dmitriy-Butramyou and @SerhiiNosko and @AndreiMPV -- You approved and merged this PR without resolving the errors that are being reported. Now every other group that has an unrelated PR will also be notified about these errors.

**Comment by SerhiiNosko** [comment]
> Attention to @Dmitriy-Butramyou and @SerhiiNosko and @AndreiMPV -- You approved and merged this PR without resolving the errors that are being reported. Now every other group that has an unrelated PR will also be notified about these errors.

Thanks for reminding, we forgot to comment about resolving and click 'Resolve' button under each comment in github, but actually all reported issues were resolved by subsequent commits before merge.

**Comment by dcrossleyau** [comment]
See the Jenkins artifact on the master branch build: https://jenkins-aws.indexdata.com/job/folio-org/job/acq-models/job/master/305/ at `API Schema Lint Report` in the left-hand panel.

**Comment by SerhiiNosko** [comment]
> See the Jenkins artifact on the master branch build: https://jenkins-aws.indexdata.com/job/folio-org/job/acq-models/job/master/305/ at `API Schema Lint Report` in the left-hand panel.

Thank you for link, we fixed this issue in the next PR:
https://jenkins-aws.indexdata.com/job/folio-org/job/acq-models/job/master/306/ApiSchemaLintReport/

**Comment by Dmitriy-Butramyou** [comment]
Hi @dcrossleyau PR with fix was merged.

**Pull request merged**